recipe/meta.yaml

```diff
@@ -97,12 +96,9 @@ requirements:
 test:
   requires:
     - python {{ python_min }}  # [unix]
-    - pip
   commands:
     - USER=test spyder -h  # [unix]
     - spyder -h  # [win]
-    # Pip fails when running but the package is installed correctly
-    - python -m pip check  # [not aarch64]
```

Comment:
does pip check fail now? 

If that is the case, can we patch spyder so that it doesn't?

Reply:
pip check will fail for the `spyder-base`, but not `spyder`. I do not know to how patch spyder so that `spyder-base` would not fail the pip check.

Reply:
You mean patch the requirements in `setup.py` just for the conda package build?

Reply:
yes

Reply:
to remove the lines that cause it to fail:
https://github.com/spyder-ide/spyder/blob/master/setup.py#L235

Reply:
Patch created. Thanks for the suggestion.